main/RFXNames.cpp

```diff
@@ -257,6 +257,7 @@ const char *Hardware_Type_Desc(int hType)
 		{ HTYPE_USBtinGateway, "USBtin Can Gateway"},
 		{ HTYPE_EnphaseAPI, "Enphase Envoy with LAN (HTTP) interface" },
 		{ HTYPE_RaspberryMCP23017, "I2C sensor GPIO 16bit expander MCP23017" },
+		{HTYPE_eHouseTCP,"eHouse UDP+TCP  - LAN Interface"},
```

Comment:
Please respect our formatting
Add a space before HTYPE, after the comma, and no double spaces and change 'LAN Interface' to 'with LAN interface'